maisim/maisim.Game/Utils/TestUtil.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Linq;
+using maisim.Game.Beatmaps;
+using maisim.Game.Scores;
+
+namespace maisim.Game.Utils
+{
+    /// <summary>
+    /// Class working as helper for create mock object using in the test.
+    /// </summary>
+    public class TestUtil
```

Comment:
I'm not so sure about using static mock beatmaps names, but i guess this can be fixed in a following PR